Source/ChocolateyGui/Models/AppConfiguration.cs

```diff
@@ -17,5 +17,7 @@ public class AppConfiguration
         public bool DefaultToTileViewForRemoteSource { get; set; }
 
         public bool UseDelayedSearch { get; set; }
+
+        public bool SearchInAllRepositories { get; set; }

```

Comment:
Is this the right terms here?  Chocolatey GUI doesn't refer to repositories, but rather Sources.  Should this be `SearchInAllRepositories`?

Reply:
I'm with @gep13 , so `SearchInAllSources` could be the name...